[2022-06-11 20:33:00,066] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: xcom_dag.is_accurate scheduled__2022-06-10T00:00:00+00:00 [queued]>
[2022-06-11 20:33:00,085] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: xcom_dag.is_accurate scheduled__2022-06-10T00:00:00+00:00 [queued]>
[2022-06-11 20:33:00,086] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 20:33:00,086] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-11 20:33:00,086] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 20:33:00,106] {taskinstance.py:1378} INFO - Executing <Task(BranchPythonOperator): is_accurate> on 2022-06-10 00:00:00+00:00
[2022-06-11 20:33:00,120] {standard_task_runner.py:52} INFO - Started process 126657 to run task
[2022-06-11 20:33:00,131] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'is_accurate', 'scheduled__2022-06-10T00:00:00+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpu33ai8fn', '--error-file', '/tmp/tmpwux7l00a']
[2022-06-11 20:33:00,134] {standard_task_runner.py:80} INFO - Job 44: Subtask is_accurate
[2022-06-11 20:33:00,244] {standard_task_runner.py:92} ERROR - Failed to execute job 44 for task is_accurate (TaskInstance for xcom_dag, is_accurate, map=-1 with run_id or execution_date of 'scheduled__2022-06-10T00:00:00+00:00' not found; 126657)
[2022-06-11 20:33:00,268] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-11 20:33:00,340] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-11 20:34:03,727] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: xcom_dag.is_accurate scheduled__2022-06-10T00:00:00+00:00 [queued]>
[2022-06-11 20:34:03,741] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: xcom_dag.is_accurate scheduled__2022-06-10T00:00:00+00:00 [queued]>
[2022-06-11 20:34:03,741] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 20:34:03,742] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-11 20:34:03,742] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 20:34:03,756] {taskinstance.py:1378} INFO - Executing <Task(BranchPythonOperator): is_accurate> on 2022-06-10 00:00:00+00:00
[2022-06-11 20:34:03,761] {standard_task_runner.py:52} INFO - Started process 127984 to run task
[2022-06-11 20:34:03,769] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'is_accurate', 'scheduled__2022-06-10T00:00:00+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpendm8eme', '--error-file', '/tmp/tmprb7audel']
[2022-06-11 20:34:03,771] {standard_task_runner.py:80} INFO - Job 56: Subtask is_accurate
[2022-06-11 20:34:03,907] {task_command.py:370} INFO - Running <TaskInstance: xcom_dag.is_accurate scheduled__2022-06-10T00:00:00+00:00 [running]> on host airflowvm
[2022-06-11 20:34:04,082] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=is_accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-06-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-10T00:00:00+00:00
[2022-06-11 20:34:04,083] {python.py:173} INFO - Done. Returned value was: None
[2022-06-11 20:34:04,084] {skipmixin.py:140} INFO - Following branch None
[2022-06-11 20:34:04,084] {skipmixin.py:175} INFO - Skipping tasks ['accurate', 'inaccurate']
[2022-06-11 20:34:04,157] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=is_accurate, execution_date=20220610T000000, start_date=20220611T203403, end_date=20220611T203404
[2022-06-11 20:34:04,201] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-11 20:34:04,274] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
